.github/workflows/R-CMD-check.yaml

```diff
@@ -3,12 +3,12 @@
 on:
   push:
     branches:
-      - main
       - master
+      - 1-15-99
```

Comment:
once the current run (https://github.com/Rdatatable/data.table/actions/runs/7938397022, https://github.com/Rdatatable/data.table/actions/runs/7938397023) passes, let's remove this carve-out for `1-15-99` from CI as well.

Reply:
`drop`'d the relevant commit & force-pushed again